.github/ISSUE_TEMPLATE/governance-review-request.yaml

```diff
@@ -1,166 +1,3 @@
-#     attributes:
-#       label: Governance documents
-#       description: |
-#         Please provide the governance documentation of your project in separate lines. This would include links to documents that explain:
-#         - Project purpose
-#         - Maintainer List (with employer information)
-#         - How your project is making decisions
-#         - How and when contributors are moving through the contributor ladder (e.g. becoming a maintainer)
-#         - Maintainer life cycle with information about how and when maintainers are demoted
-#         - Code of Conduct
-#         - Contributor Guide
-#         - How your community conduct communication and meetings
-#         - How your community handles security reporting and response
-#         - Who owns what code and docs
-        
-#         These documents do NOT need to be separate documents. Single document or a few documents that explain everything above is good for us to read, understand and evaluate your project governance.
-        
-#         If these documents are already linked from an Incubation or Graduation application, you may simply
-#         refer to that document by linking it here, and fill in only the documents not listed there.
-#     validations:
-#       required: true
-
-#   - type: textarea
-#     attributes:
-#       label: Governance Execution Examples
-#       description: |
-#         Do you have any examples of execution of the governance? Examples could be:
-#         - Voting
-#         - Promoting a contributor / demoting a maintainer
-#         - Security response
-#         - Meeting recordings
-#     validations:
-#       required: true
-
-#   - type: textarea
-#     attributes:
-#       label: Governance Evolution
-#       description: |
-#         Governance evolution is the observable changes and improvements the project makes to its governance over the project's lifespan. It is expected that changes will occur over the project's life and that such changes are iterative, tested, and adjusted.
-
-#         If there are any major milestones in the project's governance over time, please describe them here with supporting URL links.
-#     validations:
-#       required: true
-
-#   - type: markdown
-#     attributes:
-#       value: |
-#         ## Additional information
-
-#   - type: textarea
-#     attributes:
-#       label: Any specific aspects of your governance structure are you seeking feedback on?
-
-#   - type: textarea
-#     attributes:
-#       label: Do you have any concerns or specific areas where you feel your governance could be improved?
-
-#   - type: textarea
-#     attributes:
-#       label: Additional notes and resources
-#       description: Please add any additional notes and resources here.
-
-body:
-  - type: markdown
-    attributes:
-      value: |
-        This template has moved to the TOC repo. 
-        Please complete the Governance Review Request [here](https://github.com/cncf/toc/issues)
```

Comment:
Wait, maybe keep this text?  In case they ignore the description and open the issue.